Extensions/Video/JsExtension.js

```diff
@@ -0,0 +1,775 @@
+/**
+ *
+ * JsExtention.js :  Permet de déclarer les events et fait l'affichage des instances d'object dans l'éditeur de GD grace à RenderedVideoObjectInstance
+ *
+ * videoruntimeobject-pixi-renderer.js  : Gère le rendu des object dans la preview et les exports.
+ *
+ * Les actions : a déclarer dans JsExtention.js, puis ça appel dans videoruntimeobject.js, puis vers videoruntimeobject-pixi-renderer.js
+ * Si il y a un operator il faut un .setManipulatedType('number')  et     .setGetter('getVolume');
+ * la nouvelle valeur calculé avec l'opérateur est envoyé directement à setFunctionName("setVolume") !
+ *
+ *
+ */
+
+/**
+ * This is a declaration of an extension for GDevelop 5.
+ *
+ * ℹ️ Run `node import-GDJS-Runtime.js` (in newIDE/app/scripts) if you make any change
+ * to this extension file or to any other *.js file that you reference inside.
+ *
+ * The file must be named "JsExtension.js", otherwise GDevelop won't load it.
+ * ⚠️ If you make a change and the extension is not loaded, open the developer console
+ * and search for any errors.
+ *
+ * More information on https://github.com/4ian/GDevelop/blob/master/newIDE/README-extensions.md
+ */
+module.exports = {
+  createExtension: function(_, gd) {
+    const extension = new gd.PlatformExtension();
+    extension
+      .setExtensionInformation(
+        "Video",
+        "Video",
+        "Display a video in your scene",
```

Comment:
Let's avoid speaking to the user directly and keep description generic. Also add the translation function:
`_("Display a video on the scene")`